movie_search/tmdb_api.py

```diff
@@ -8,26 +8,25 @@
 import json
 from django.conf import settings
 
-BASE_URL = "https://api.themoviedb.org/3"
-LANG_ENG = "en-US"
 
+class TMDBApi:
```

Comment:
I like this grouping into a class, similar to what I did here: https://www.youtube.com/watch?v=btBFH5GKd-k